spec/requests/api/user_can_see_list_of_posts_spec.rb

```diff
@@ -0,0 +1,28 @@
+RSpec.describe 'GET /api/posts', type: :request do
+  let!(:post) { 3.times { create(:post) } }
```

Comment:
do you really need the bang(!) ?

Reply:
Yes, because we are not calling it in the test